### ISSUE TITLE
Quick Panel `quick_panel_path_label` link_color property

### ISSUE THREAD

**Issue by TerminalFi** [meetio-theme/sublime-meetio-theme#46]
**Describe the bug**
ST 4083+ introduced mini_html inside of the quick panel. Now we have links that are clickable.

**To Reproduce**
1. Package Control: Install Packages
2. Scroll down to

**Expected behavior**
A clean and clear color blue for links

**Screenshots**

![image](https://user-images.githubusercontent.com/32599364/91205963-6890d980-e6d4-11ea-8a21-4b3e5c3b6247.png)

**Comment by mauroreisvieira** [comment]
Thanks @TheSecEng for reporting that, I will try to solve that.

**Comment by TerminalFi** [comment]
It just requires that `quick_panel_path_label` get a `link_property`  Tested locally and it worked.

**Comment by mauroreisvieira** [comment]
@TheSecEng can you provide a snippet code to see?

**Comment by TerminalFi** [comment]
Sure, this is in my override

```json
        {
            "class": "quick_panel_path_label",
            "link_color": "var(colorAccent)"
        },
```

Which results in a nice look

![image](https://user-images.githubusercontent.com/32599364/91296345-7729d000-e76a-11ea-87c4-d28a1c680546.png)

**Comment by mauroreisvieira** [comment]
I can't reproduce that :( 

![Screenshot 2020-08-26 at 12 18 52](https://user-images.githubusercontent.com/3084774/91297502-56776f80-e796-11ea-9b15-547a817764a3.png)

**Comment by TerminalFi** [comment]
It’s not released in PC yet. But it’s in master

**Comment by mauroreisvieira** [comment]
Ok. Let's wait for release.

**Comment by abathur** [comment]
Theme updated today after my first reboot in a few weeks. Noticed foreground color in some inputs/quick panel was off. I am not sure this is the cause, but I did open my console to a few thousand of these:

```
no such prop link_color when matching quick_panel_path_label against quick_panel_path_label
```

Dev channel build 3210.

**Comment by mauroreisvieira** [comment]
Hi @abathur, this error in you console it's expected because `link_color` it's a feature of ST4.

But I will try to fix that. 

Thanks for your report.

**Comment by abathur** [comment]
@mauroreisvieira Yep--I'm aware. 

If it's just a warning it's not a huge deal (though the message is pretty spammy--I had over 2300 lines in my console), but I only *noticed* it because something broke my input foreground color, and I went looking for clues. I ran out of energy to try and fix the issue--I'll just live with the break for a while--but I wanted to mention it in case it's having other effects.
